### PR TITLE
docker repo migrated

### PR DESCRIPTION
**Pull request by BLaZeKiLL**
#### Changes

- updated docker repo in image-tag.js
- hard coded docker repo version as `0`
- hard coded platform as `base`
- updated image-tag.test.js
- updated docker.test.js to bypass repo version issue

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [ ] Readme (updated or not needed)
- [x] Tests (added, updated or not needed)

### CODE REVIEW

File: action/steps/run_tests.sh [hunk 1]
```diff
@@ -74,19 +74,18 @@ if [ $EDIT_MODE = true ]; then
...
-      -logFile "$FULL_ARTIFACTS_PATH/editmode.log" \
-      -projectPath "$UNITY_PROJECT_PATH" \
-      -runTests \
-      -testPlatform editmode \
-      -testResults "$FULL_ARTIFACTS_PATH/editmode-results.xml" \
-      $CUSTOM_PARAMETERS
-      
+  unity-editor \
+    -batchmode \
+    -logFile "$FULL_ARTIFACTS_PATH/editmode.log" \
```
Comment (BLaZeKiLL) [inline action/steps/run_tests.sh:79]: `-nographics` flag needs to be added here also one more place in the same file, should that be a separate PR ?

**Comment by BLaZeKiLL** [comment]
I don't know much about unity tests but can the user write build platform dependant tests ? in that case we would need to handle build platform like in the unity-builder action

**Comment by sbergen** [comment]
I can confirm that this fixes at least the issue of missing `git lfs` when restoring packages on 2020.1 images :+1:

**Comment by sbergen** [comment]
> I don't know much about unity tests but can the user write build platform dependant tests ? in that case we would need to handle build platform like in the unity-builder action

You can definitely use things like `#if UNITY_ANDROID` in code, and want to run tests on specific platforms. There's also the `UnityPlatform` attribute, which can be used to target tests at specific platforms. I'd see this as a separate issue from what is done in this PR, though :)

File: action/steps/run_tests.sh (hunk 1 above)
Comment (webbertakken) [reply action/steps/run_tests.sh:79]: Up to you. I'm fine with either.

**Comment by webbertakken** [comment]
I'll merge this after you regenerate `action/index.js` after rebasing or merging main into this.
